runtime/command-start/src/main/java/io/aklivity/zilla/runtime/command/start/internal/airline/ZillaStartCommand.java

```diff
@@ -142,18 +142,19 @@ public void run()
             }
         }
 
-        final Consumer<Throwable> report = exceptions
+        final Consumer<Throwable> reporter = exceptions
             ? e -> e.printStackTrace(System.err)
             : e -> System.err.println(e.getMessage());
 
         final ErrorHandler onError = ex ->
         {
-            report.accept(ex);
+            reporter.accept(ex);
             stop.countDown();
         };
```

Comment:
There is still code duplication between `zilla start` here and the default `reporter` in `EngineBuilder`.

If we omit the `reporter` then it will default.

Perhaps it should default to reporting the exception and then triggering the error handler by default when onError would otherwise be called, so that we are not forced to explicitly create the reported here just to chain with the error handler.

We should also move the defaulting logic to `EngineConfiguration`. For that to work, we'll need to create an inner functional interface in `EngineConfiguration`, something like `ErrorReporter` which has `void report(Throwable ex)` or similar and default method that returns a different value depending on `exceptions` configuraiton setting in `EngineConfiguration`.